### ISSUE TITLE
Zulip setup hangs when setting `DISABLE_HTTPS: "True"`

### ISSUE THREAD

**Issue by thomascft** [zulip/docker-zulip#362]
When setting the debug flag it looks like it stops at `/home/zulip/deployments/current/scripts/zulip-puppet-apply -f` right after `Disabling https in nginx.`

**Comment by maltokyo** [comment]
Please share the relevant docker-compose section, I can take a look, as I disable HTTPS without issue.

I did it using this help: https://github.com/zulip/docker-zulip/issues/229

**Comment by CharlesChiuGit** [comment]
@maltokyo Where exactly should I place `DISABLE_HTTPS: "True"`. I tried put it in `docker-compose.yml` -> `zulip:` -> `environment:`, but it still prompt me to use `https://zulip.example.com`. 

If I comment out `DISABLE_HTTPS: "True"`, I can access zulip without any weird layout.

Here's part of my `docker-compose.yml`:

```yml
zulip:
    image: "zulip/docker-zulip:6.1-0"
    restart: unless-stopped
    build:
      context: .
      args:
        # Change these if you want to build zulip from a different repo/branch
        ZULIP_GIT_URL: https://github.com/zulip/zulip.git
        ZULIP_GIT_REF: "6.1"
        # Set this up if you plan to use your own CA certificate bundle for building
        # CUSTOM_CA_CERTIFICATES:
    ports:
      - "7777:80"
      - "443:443"
    environment:
      # DISABLE_HTTPS: "True"
      DB_HOST: "database"
      DB_HOST_PORT: "5432"
      DB_USER: "zulip"
      SSL_CERTIFICATE_GENERATION: "self-signed"
      SETTING_MEMCACHED_LOCATION: "memcached:11211"
      SETTING_RABBITMQ_HOST: "rabbitmq"
      SETTING_REDIS_HOST: "redis"
```

**Comment by maltokyo** [comment]
That is how it is for me:
(I also comment out the `ports` section, as it is not needed when running behind a reverse proxy)
```
    environment:
      #FORCE_FIRST_START_INIT: 'True'
      DB_HOST: 'zulip_postgresql'
      DB_HOST_PORT: '5432'
      DB_USER: 'zulip'
      DISABLE_HTTPS: 'True'
.
.
.
```

**Comment by thomascft** [comment]
I found a solution a bit ago, and thought I'd post it here in case anyone else has the same issue. Randomly things would just not work, this isn't exclusive to zulip either. I could setup and run certain containers on my personal machine, but when it came to deploying them to the development server they just wouldn't work. My problems had to do with the fact that I was using the snap version of docker that was preinstalled on Ubuntu. Once I replaced the snap version with the native version all my problems vanished.
